bagit.xml

```diff
@@ -287,8 +287,7 @@ The base directory can have any name.
       |
       +-- [optional tag directories]/
             |
-            +-- [optional tag files]
-  </artwork>
+            +-- [optional tag files] </artwork>
```

Comment:
Was the intention of this to avoid an extra line in the rendered text output? I'm not a huge fan of the closing tag being on the end of the line like this but I'm not sure it's worth changing everything to go the other way.

Reply:
Exactly, and it definitely makes the XML ugly. It may be a defect in the xml2rfc tool. If you can find a way around it, go for it, but if push comes to shove, I think getting the rendered human-oriented document consistent and correct is more important than making the XML pretty.